data/multiplayer/scenarios/4p_King_of_the_Hill.cfg

```diff
@@ -4,20 +4,44 @@
     [store_side]
         side={SIDE}
     [/store_side]
-    {RANDOM $side.recruit}
-    [store_unit_type]
-        type=$random
-    [/store_unit_type]
-    # If the unit is level 0 then switch to one of its advancements instead.
-    [if]
-        [variable]
-            name=unit_type.level
-            equals=0
-        [/variable]
-        [then]
-            {RANDOM $unit_type.advances_to}
-        [/then]
-    [/if]
+    # Get the faction leader list of the side.
+    [lua]
+        code=<<
+            local t = ...
+            local era = wesnoth.get_era(wesnoth.scenario.era.id)
```

Comment:
I forgot this existed… I think maybe it will work to just use `wesnoth.scenario.era` and not call `wesnoth.get_era`?

Reply:
You are correct, it works. Updated.